.gitignore

```diff
@@ -1 +1,2 @@
 /target
+/http-cacache
```

Comment:
Could: Comment explaining that this is not a typo

Reply:
https://github.com/zkat/cacache-rs

Looks like it's the name of the underlying caching implementation